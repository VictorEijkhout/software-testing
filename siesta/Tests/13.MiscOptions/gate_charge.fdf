%include ../base_wat_bigbox.fdf
SystemLabel     gate_charge

Slab.DipoleCorrection True
%block Geometry.Charge
 plane -0.001 charge
  gauss 1. 2. Ang
   0. 20. 0. Ang
   0.  1. 0.
%endblock Geometry.Charge

