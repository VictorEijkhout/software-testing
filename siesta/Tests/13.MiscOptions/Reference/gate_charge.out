Siesta Version  : 5.0.0-alpha-123-gfb8fd7d02
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
GEMM3M support
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/13.MiscOptions/work/gate_charge
* Running in serial mode.
>> Start of run:  28-SEP-2023  15:40:04

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.97704
************************** Dump of input data file ****************************
%include ../../basejob.fdf
SystemLabel     gate_charge
Slab.DipoleCorrection True
%block Geometry.Charge
 plane -0.001 charge
  gauss 1. 2. Ang
   0. 20. 0. Ang
   0.  1. 0.
%endblock Geometry.Charge
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: 2 Water molecules
reinit: -----------------------------------------------------------------------
reinit: System Label: gate_charge
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:    8 Label: O
Species number:   2 Atomic number:    1 Label: H


---- Processing specs for species: O
Ground state valence configuration:   2s02  2p04
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/O.psf

---- Processing specs for species: H
Ground state valence configuration:   1s01
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/H.psf

---- Pseudopotential check for O 

Pseudized shells:
2s( 2.00) rc: 1.14
2p( 4.00) rc: 1.14
3d( 0.00) rc: 1.14
4f( 0.00) rc: 1.14
Valence configuration for ps generation: (assumed as above)

---- Pseudopotential check for H 

Pseudized shells:
1s( 1.00) rc: 1.25
2p( 0.00) rc: 1.25
3d( 0.00) rc: 1.25
4f( 0.00) rc: 1.25
Valence configuration for ps generation: (assumed as above)
For O, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
For H, standard SIESTA heuristics set lmxkb to 2
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
O                    Z=   8    Mass=  15.999        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=0  (2s)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=1  (2p)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=0  polorb=0  (3d)  (perturbative polarization orbital) (from 2p)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for O                     (Z =   8)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    6.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.1366
V l=1 = -2*Zval/r beyond r=  1.1332
V l=2 = -2*Zval/r beyond r=  1.1298
V l=3 = -2*Zval/r beyond r=  1.1264
All V_l potentials equal beyond r=  1.1355
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.1366
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     33.667 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     76.727 Ry
atom: Maximum radius for chlocal:    1.37241
atom: Maximum radius for r*vlocal+2*Zval:    1.25469

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.293209   el= -1.742414   Ekb=  9.076872   kbcos=  0.325320
GHOST: No ghost state for L =  1
   l= 1   rc=  1.284119   el= -0.676589   Ekb= -8.194485   kbcos= -0.396010
GHOST: No ghost state for L =  2
   l= 2   rc=  1.433152   el=  0.013307   Ekb= -2.071665   kbcos= -0.003690
GHOST: No ghost state for L =  3
   l= 3   rc=  1.540891   el=  0.019584   Ekb= -0.817137   kbcos= -0.000371

KBgen: Total number of Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    3.560943
                 energy =   -1.731321
                kinetic =    1.587329
    potential(screened) =   -3.318651
       potential(ionic) =  -11.260325

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    4.290924
                 energy =   -0.665565
                kinetic =    4.934810
    potential(screened) =   -5.600376
       potential(ionic) =  -13.346429

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 2p
Split based on tail norm

   izeta = 1
                     rc =    4.290924
                 energy =    2.236783
                kinetic =    4.278617
    potential(screened) =   -2.041834
       potential(ionic) =   -8.171412
atom: Total number of Sankey-type orbitals:  9

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 4.00)                                                            
 3d( 0.00)                                                            
Vna: chval, zval:    6.00000   6.00000

Vna:  Cut-off radius for the neutral-atom potential:   4.290924

atom: _________________________________________________________________________

<basis_specs>
===============================================================================
H                    Z=   1    Mass=  1.0080        Charge= 0.17977+309
Lmxo=1 Lmxkb= 2    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=1
          i=1  nzeta=1  polorb=1  (1s)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=0  polorb=0  (2p)  (perturbative polarization orbital) (from 1s)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for H                     (Z =   1)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    1.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.2384
V l=1 = -2*Zval/r beyond r=  1.2321
V l=2 = -2*Zval/r beyond r=  1.2272
All V_l potentials equal beyond r=  1.2371
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.2384
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     28.363 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     64.639 Ry
atom: Maximum radius for chlocal:    1.44620
atom: Maximum radius for r*vlocal+2*Zval:    1.31026

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.364142   el= -0.467325   Ekb= -2.009475   kbcos= -0.336680
GHOST: No ghost state for L =  1
   l= 1   rc=  1.430269   el=  0.007562   Ekb= -0.503817   kbcos= -0.022554
GHOST: No ghost state for L =  2
   l= 2   rc=  1.469686   el=  0.013286   Ekb= -0.191752   kbcos= -0.002430

KBgen: Total number of Kleinman-Bylander projectors:   9
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 1s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    5.285354
                 energy =   -0.456120
                kinetic =    0.900445
    potential(screened) =   -1.356565
       potential(ionic) =   -1.890080

POLgen: Perturbative polarization orbital with L=  1

POLgen: Polarization orbital for state 1s
Split based on tail norm

   izeta = 1
                     rc =    5.285354
                 energy =    0.642323
                kinetic =    1.282472
    potential(screened) =   -0.640149
       potential(ionic) =   -1.117275
atom: Total number of Sankey-type orbitals:  4

atm_pop: Valence configuration (for local Pseudopot. screening):
 1s( 1.00)                                                            
 2p( 0.00)                                                            
Vna: chval, zval:    1.00000   1.00000

Vna:  Cut-off radius for the neutral-atom potential:   5.285354

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1    8 O                       # Species index, atomic number, species label
    2    1 H                       # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
O                     2                    # Species label, number of l-shells
 n=2   0   1                         # n, l, Nzeta 
   3.561   
   1.000   
 n=2   1   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   4.291   
   1.000   
H                     1                    # Species label, number of l-shells
 n=1   0   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   5.285   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file O.ion.nc
Dumping basis to NetCDF file H.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      1.43052   1.10738   0.00000  2        2
siesta:     -1.43052   1.10738   0.00000  2        3
siesta:      0.00000   9.44863   0.00000  1        4
siesta:      1.43052   8.34125   0.00000  2        5
siesta:     -1.43052   8.34125   0.00000  2        6

siesta: System type = slab      

initatomlists: Number of atoms, orbitals, and projectors:      6    34    68

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        2
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   300.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     1000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   299.9863 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 2
mix.SCF:    Linear mixing weight                 =     0.250000
mix.SCF:    Mixing weight                        =     0.250000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2500
  weight.linear 0.2500
  history 2
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    16.001000
Total ionic charge:    16.000000
madelung: WARNING: Charged system, but not an atom or molecule.
madelung: WARNING: Energy correction terms can not be applied. 
madelung: WARNING: Continue only if you really know what you are doing.

siesta: k-grid: Number of k-points = 1
siesta: k-point is Gamma-only

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

outcell: Unit cell vectors (Ang):
        3.000000    0.000000    0.000000
        0.000000   30.000000    0.000000
        0.000000    0.000000    5.000000

outcell: Cell vector modules (Ang)   :    3.000000   30.000000    5.000000
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :    450.0000
Gamma-point calculation with interaction between periodic images
Some features might not work optimally:
e.g. DM initialization from atomic data
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=34 nrows=34 sparsity=.8599 nnzs=994, refcount: 7>
  <dData1D:(new from dSpData1D) n=994, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=34 nrows=34 sparsity=.8599 nnzs=994, refcount: 8>
  <dData2D:DM n=994 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       6      41

InitMesh: MESH = 32 x 320 x 54 = 552960
InitMesh: Mesh cutoff (required, used) =   300.000   314.456 Ry
New grid distribution [1]: sub = 2

Geometric charge distributions:
   Infinite Gauss-plane   adding -0.10000E-02 e divided in 74304 grid points


dipole: A dipole layer will be introduced in the vacuum
dipole: region to compensate the system dipole
Dipole moment in unit cell =      -0.0000      -0.0602      -0.0000 D
Electric field for dipole correction =     0.000000     0.005044     0.000000 eV/Ang/e

stepf: Fermi-Dirac step function
Dipole moment in unit cell =      -0.0000      -0.0540      -0.0000 D
Electric field for dipole correction =     0.000000     0.004523     0.000000 eV/Ang/e

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -218.238147
siesta: Eions   =      1621.026762
siesta: Ena     =       347.677440
siesta: Ekin    =       739.519077
siesta: Enl     =      -137.074556
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =       -28.406869
siesta: DUscf   =         4.863409
siesta: DUext   =         0.000025
siesta: Ex      =      -205.186526
siesta: Ec      =       -25.401422
siesta: Exc     =      -230.587948
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -916.587733
siesta: Etot    =      -925.036184
siesta: FreeEng =      -925.036406

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -916.587733     -925.036184     -925.036406  0.606751 -2.196966  3.276375
timer: Routine,Calls,Time,% = IterSCF        1       0.071   8.47
Dipole moment in unit cell =      -0.0000      -0.0620      -0.0000 D
Electric field for dipole correction =     0.000000     0.005192     0.000000 eV/Ang/e
   scf:    2     -890.781362     -915.706866     -915.707089  0.667890 -3.459892  7.626992
Dipole moment in unit cell =      -0.0000      -0.0557      -0.0000 D
Electric field for dipole correction =     0.000000     0.004662     0.000000 eV/Ang/e
   scf:    3     -929.186498     -926.901978     -926.902201  0.507440 -2.792003  0.501998
Dipole moment in unit cell =      -0.0000      -0.0563      -0.0000 D
Electric field for dipole correction =     0.000000     0.004712     0.000000 eV/Ang/e
   scf:    4     -926.968642     -926.946056     -926.946278  0.025241 -2.945089  0.198958
Dipole moment in unit cell =      -0.0000      -0.0565      -0.0000 D
Electric field for dipole correction =     0.000000     0.004734     0.000000 eV/Ang/e
   scf:    5     -926.950536     -926.948732     -926.948954  0.004848 -3.050413  0.102970
Dipole moment in unit cell =      -0.0000      -0.0566      -0.0000 D
Electric field for dipole correction =     0.000000     0.004739     0.000000 eV/Ang/e
   scf:    6     -926.950575     -926.949685     -926.949907  0.002806 -3.123374  0.039433
Dipole moment in unit cell =      -0.0000      -0.0566      -0.0000 D
Electric field for dipole correction =     0.000000     0.004739     0.000000 eV/Ang/e
   scf:    7     -926.950158     -926.949936     -926.950158  0.001536 -3.160064  0.017340
Dipole moment in unit cell =      -0.0000      -0.0566      -0.0000 D
Electric field for dipole correction =     0.000000     0.004738     0.000000 eV/Ang/e
   scf:    8     -926.950003     -926.949970     -926.950192  0.000400 -3.169238  0.011143
Dipole moment in unit cell =      -0.0000      -0.0566      -0.0000 D
Electric field for dipole correction =     0.000000     0.004738     0.000000 eV/Ang/e
   scf:    9     -926.949997     -926.949984     -926.950206  0.000463 -3.179398  0.004130
Dipole moment in unit cell =      -0.0000      -0.0566      -0.0000 D
Electric field for dipole correction =     0.000000     0.004738     0.000000 eV/Ang/e
   scf:   10     -926.949984     -926.949984     -926.950206  0.000033 -3.179320  0.002571
Dipole moment in unit cell =      -0.0000      -0.0566      -0.0000 D
Electric field for dipole correction =     0.000000     0.004738     0.000000 eV/Ang/e
   scf:   11     -926.949984     -926.949984     -926.950207  0.000036 -3.178713  0.000904

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000362445
max |H_out - H_in|      (eV) :     0.0009040142
SCF cycle converged after 11 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       6      41
Dipole moment in unit cell =      -0.0000      -0.0566      -0.0000 D
Electric field for dipole correction =     0.000000     0.004738     0.000000 eV/Ang/e

siesta: E_KS(eV) =             -926.9500

siesta: E_KS - E_eggbox =      -926.9500

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot   -0.000000   -0.002145   -0.000000
----------------------------------------
   Max    2.534551
   Res    1.054077    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    2.534551    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):      -51.05      -10.21       -0.50       -0.00       -0.00       -0.00
(Free)E + p*V (eV/cell)     -921.1682
Target enthalpy (eV/cell)     -926.9502

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -243.833410
siesta: Eions   =      1621.026762
siesta: Ena     =       347.677440
siesta: Ekin    =       695.865997
siesta: Enl     =      -124.675639
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        -3.048225
siesta: DUscf   =         1.560792
siesta: DUext   =         0.000028
siesta: Ex      =      -198.236453
siesta: Ec      =       -25.067161
siesta: Exc     =      -223.303614
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -926.949984
siesta: Etot    =      -926.949984
siesta: FreeEng =      -926.950207

siesta: Final energy (eV):
siesta:  Band Struct. =    -243.833410
siesta:       Kinetic =     695.865997
siesta:       Hartree =    2603.388927
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000028
siesta:         Exch. =    -198.236453
siesta:         Corr. =     -25.067161
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -223.303614
siesta:  Ion-electron =   -5853.917208
siesta:       Ion-ion =    1851.015886
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =    -926.949984
siesta:         Fermi =      -3.178713

siesta: Atomic forces (eV/Ang):
siesta:      1   -0.000000   -2.534551    0.000000
siesta:      2    0.390226    1.279930   -0.000000
siesta:      3   -0.390226    1.279930   -0.000000
siesta:      4   -0.000000    2.531842    0.000000
siesta:      5    0.393212   -1.279648   -0.000000
siesta:      6   -0.393212   -1.279648   -0.000000
siesta: ----------------------------------------
siesta:    Tot   -0.000000   -0.002145   -0.000000

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.031861   -0.000000    0.000000
siesta:    -0.000000   -0.006374   -0.000000
siesta:    -0.000000   -0.000000   -0.000312

siesta: Cell volume =        450.000000 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00013994          0.00010724  Ry/Bohr**3
siesta:           0.01284895          0.00984596  eV/Ang**3
siesta:          20.58628785         15.77496117  kBar
(Free)E+ p_basis*V_orbitals  =        -925.720153
(Free)Eharris+ p_basis*V_orbitals  =        -925.720153

siesta: Electric dipole (a.u.)  =   -0.000000   -0.022265   -0.000000
siesta: Electric dipole (Debye) =   -0.000000   -0.056593   -0.000000

cite: Please see "gate_charge.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-alpha-123-gfb8fd7d02
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302
        Slab-dipole correction
          DOI: www.doi.org/10.1103/PhysRevB.59.12301
        Charge/Hartree gate model
          DOI: www.doi.org/10.1039/C5CP04613K


timer: Elapsed wall time (sec) =       1.908
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       1.901       1.901   100.00
Setup                  1       0.064       0.064     3.37
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       1.837       1.837    96.61
geom_init              1       0.704       0.704    37.04
state_init             1       0.048       0.048     2.53
hsparse                1       0.000       0.000     0.02
overlap                1       0.048       0.048     2.50
Setup_H0               1       0.587       0.587    30.85
naefs                  2       0.000       0.000     0.01
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.023       0.047     2.45
nlefsm                 2       0.202       0.403    21.20
DHSCF_Init             1       0.139       0.139     7.29
DHSCF1                 1       0.016       0.016     0.84
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.122       0.122     6.44
REMESH                 1       0.007       0.007     0.39
REORD                 34       0.000       0.016     0.84
PHION                  1       0.084       0.084     4.40
COMM_BSC              55       0.000       0.020     1.05
POISON                14       0.013       0.187     9.81
fft                   28       0.005       0.133     6.98
setup_H               12       0.071       0.849    44.63
DHSCF                 13       0.092       1.194    62.80
DHSCF3                13       0.071       0.917    48.22
rhoofd                13       0.019       0.244    12.85
XC                    13       0.014       0.183     9.60
GXC-CellXC            13       0.013       0.163     8.55
gridxc@cellXC         13       0.013       0.163     8.55
vmat                  13       0.013       0.169     8.86
IterSCF               11       0.071       0.783    41.16
compute_dm            11       0.000       0.002     0.12
diagon                11       0.000       0.002     0.11
r-eigvec              11       0.000       0.002     0.09
r-buildHS             11       0.000       0.000     0.00
rdiag                 11       0.000       0.002     0.09
rdiag1                11       0.000       0.000     0.02
rdiag2                11       0.000       0.000     0.01
rdiag3                11       0.000       0.001     0.06
rdiag4                11       0.000       0.000     0.00
r-buildD              11       0.000       0.000     0.01
MIXER                 10       0.000       0.000     0.01
PostSCF                1       0.350       0.350    18.40
DHSCF4                 1       0.277       0.277    14.58
dfscf                  1       0.256       0.256    13.45
overfsm                1       0.001       0.001     0.03
state_analysis         1       0.000       0.000     0.01
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.000       0.000     0.01
optical                1       0.000       0.000     0.00
  

>> End of run:  28-SEP-2023  15:40:06
Job completed
